concepticondata/conceptlists/Davies-1985-116.tsv

```diff
@@ -0,0 +1,117 @@
+ID	NUMBER	ENGLISH	CONCEPTICON_ID	CONCEPTICON_GLOSS
+Davies-1985-116-1	1	hair	1040	HAIR
+Davies-1985-116-2	2	head	1256	HEAD
+Davies-1985-116-3	3	mouth	674	MOUTH
+Davies-1985-116-4	4	nose	1221	NOSE
+Davies-1985-116-5	5	eye	1248	EYE
+Davies-1985-116-6	6	neck	1333	NECK
+Davies-1985-116-7	7	belly	1251	BELLY
+Davies-1985-116-8	8	skin	763	SKIN
+Davies-1985-116-9	9	man	1554	MAN
+Davies-1985-116-10	10	woman	962	WOMAN
+Davies-1985-116-11	11	bird	937	BIRD
+Davies-1985-116-12	12	dog	2009	DOG
+Davies-1985-116-13	13	he bites	1403	BITE
+Davies-1985-116-14	14	he sits	1416	SIT
+Davies-1985-116-15	15	he stands	1442	STAND
+Davies-1985-116-16	16	road	2457	PATH OR ROAD
```

Comment:
Maybe use 667 ROAD, as PATH OR ROAD is a bit broader?

Reply:
done.